multiples.py

```diff
@@ -15,6 +16,14 @@ def good(i):
     return by_3 or by_5
 
 
+def even_elements_mult(i):
```

Comment:
Bad name for the variable, the name should explain the nature of what is it, I suggest something like `limit`